docs/reference/commandline/trust_signer_add.md

```diff
@@ -0,0 +1,213 @@
+v1                  74d4bfa917d55d53c7df3d2ab20a8d926874d61c3da5ef6de15dd2654fc467c4   bob
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo:
+Repository Key:	ecc457614c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4555b3c6ab02f71e
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+```bash
+$ docker trust inspect example/trust-demo2
+SIGNED TAG          DIGEST                                                             SIGNERS
+v1                  74d4bfa917d55d53c7df3d2ab20a8d926874d61c3da5ef6de15dd2654fc467c4   bob
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo2:
+Repository Key:	ece554f14c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4553d2ab20a8d9268
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+Add `alice` to both repositories with a single `docker trust signer-add` command:
+
+```bash
+$ docker trust signer-add alice example/trust-demo example/trust-demo2 -k alice.crt
+
+Adding signer "alice" to example/trust-demo...
+Enter passphrase for repository key with ID 95b9e55: 
+Successfully added signer: alice to example/trust-demo
+
+Adding signer "alice" to example/trust-demo2...
+Enter passphrase for repository key with ID ece554f: 
+Successfully added signer: alice to example/trust-demo2
+```
+`docker trust inspect` now lists `alice` as a valid signer of both `example/trust-demo` and `example/trust-demo2`:
+
+
+```bash
+$ docker trust inspect example/trust-demo
+SIGNED TAG          DIGEST                                                             SIGNERS
+v1                  74d4bfa917d55d53c7df3d2ab20a8d926874d61c3da5ef6de15dd2654fc467c4   bob
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+alice               05e87edcaecb
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo:
+Repository Key:	95b9e5514c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4555b3c6ab02f71e
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+```bash
+$ docker trust inspect example/trust-demo2
+SIGNED TAG          DIGEST                                                             SIGNERS
+v1                  74d4bfa917d55d53c7df3d2ab20a8d926874d61c3da5ef6de15dd2654fc467c4   bob
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+alice               05e87edcaecb
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo2:
+Repository Key:	ece554f14c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4553d2ab20a8d9268
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+
+## Partial success
+
+With `docker trust signer-add` failure to add a signer to a repository in the list does not affect the adding of a signer to any other repositories specified:
```

Comment:
how about:

"`docker trust signer-add` will add signers to repositories on a best effort basis, so it will continue to add the signer to subsequent repositories if one attempt fails:"

And then cut the text starting with "Having the..." below?